src/components/voting/votingComponent.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Table, TableHead, TableRow, TableCell } from 'react-toolbox/lib/table';
+import { listAccountDelegates, listDelegates } from '../../utils/api/delegate';
+import VotingHeader from './votingHeader';
+import styles from './voting.css';
+
+class VotingComponent extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      delegates: [],
+      selected: [],
+      query: '',
+    };
+  }
+  componentDidMount() {
+    listAccountDelegates(this.props.activePeer, this.props.address).then((res) => {
+      this.setState({
+        delegates: res.delegates,
+      });
+    });
+  }
+  search(value) {
+    listDelegates(this.props.activePeer, { q: value }).then((res) => {
+      this.setState({
+        delegates: res.delegates,
+      });
+    });
+  }
+  handleRowSelect(selected) {
+    // this.setState({ selected: selected.map(item => sortedData[item].name) });
+    this.setState({
+      selected: selected.map(item => this.state.delegates[item].username),
+    });
+  }
+  render() {
+    return (
+      <div className="box">
+        <VotingHeader search={ value => this.search(value) }></VotingHeader>
+        <Table className={styles.table}
```

Comment:
The table should spread the whole width of the containing card. Especially selected rows look bad. The same thing was solved In Forged Blocks: https://github.com/LiskHQ/lisk-nano/blob/development/src/components/forging/forgedBlocks.js#L16